cli/flags/common.go

```diff
@@ -21,6 +21,18 @@ const (
 	DefaultCertFile = "cert.pem"
 	// FlagTLSVerify is the flag name for the TLS verification option
 	FlagTLSVerify = "tlsverify"
+	// FormatHelp describes the --format flag behavior for list commands
+	FormatHelp = `Format output using a custom template:
+'table':            Print output in table format with column headers (default)
+'table <TEMPLATE>': Print output in table format using the given Go template
```

Comment:
Wondering if this works well with our markdown and html rendering (wondering if the `<>` would be hidden in the generated HTML (something we should check 😅).

Otherwise we could also consider to make it `TEMPLATE` (without brackets)